diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatExtension.kt

```diff
@@ -0,0 +1,31 @@
+package org.cqfn.diktat.plugin.gradle
+
+import com.pinterest.ktlint.core.Reporter
+import org.gradle.api.file.FileCollection
+
+/**
+ * An extension to configure diktat in build.gradle(.kts) file
+ */
+open class DiktatExtension {
```

Comment:
data class mb?

Reply:
This class has to be open to work with gradle, and data class is always final.